pf9/cluster/helpers.py

```diff
@@ -1,5 +1,6 @@
 import re
 import netifaces
+import ipaddress
```

Comment:
The installer should change to include the ipaddress module

Reply:
From what I can tell it is part of 2.7.6 already https://cpython-test-docs.readthedocs.io/en/latest/howto/ipaddress.html

Reply:
>>> import ipaddress
>>> dir(ipaddress)
['AddressValueError', 'IPV4LENGTH', 'IPV6LENGTH', 'IPv4Address', 'IPv4Interface', 'IPv4Network', 'IPv6Address', 'IPv6Interface', 'IPv6Network', 'NetmaskValueError', '_BaseAddress', '_BaseNetwork', '_BaseV4', '_BaseV6', '_IPAddressBase', '_IPv4Constants', '_IPv6Constants', '_TotalOrderingMixin', '__builtins__', '__doc__', '__file__', '__name__', '__package__', '__version__', '_collapse_addresses_internal', '_compat_bit_length', '_compat_bytes_to_byte_vals', '_compat_int_from_byte_vals', '_compat_int_types', '_compat_range', '_compat_str', '_compat_to_bytes', '_count_righthand_zero_bits', '_find_address_range', '_split_optional_netmask', 'collapse_addresses', 'get_mixed_type_key', 'ip_address', 'ip_interface', 'ip_network', 'itertools', 'struct', 'summarize_address_range', 'unicode_literals', 'v4_int_to_packed', 'v6_int_to_packed']
>>> ipaddress.__file__
'/home/centos/pf9/pf9-venv/lib/python2.7/site-packages/ipaddress.pyc'